src/main/content/_drafts/2018-09-12-build-and-push-spring-boot-docker-images.adoc

```diff
@@ -0,0 +1,176 @@
+build and push efficient Docker images without the hassle of writing Dockerfiles.
+
+**Note** : The boost-maven-plugin is in its beta stage. 
+
+== Steps to configure Boost Maven plugin
+
+You can configure the `boost-maven-plugin` in the `plugins` section of the pom.xml
+of your Spring Boot application as shown below:
+----
+<plugin>
+  <groupId>io.openliberty.boost</groupId>
+  <artifactId>boost-maven-plugin</artifactId>
+  <version>0.1</version>
+</plugin>
+----
+
+It requires minimal knowledge of Docker including installing and running Docker on your machine.
+
+NOTE: `spring-boot-maven-plugin` must be configured before this plugin in order to create the required executable archive
+of the application.
+
+
+=== Build Docker image
+
+To build a docker image you can add execution goal to the plugin inside pom.xml like:
+----
+<plugin>
+  <groupId>io.openliberty.boost</groupId>
+  <artifactId>boost-maven-plugin</artifactId>
+  <version>0.1</version>
+  <executions>
+    <execution>
+      <goals>
+        <goal>docker-build</goal>
+      </goals>
+    </execution>
+  </executions>
+</plugin>
+----
+`mvn clean package` will create a docker image with a default repository name as `${project.artifactId}`
+ and a default tag `latest`.
+
+Alternatively, you can configure the plugin with no execution goals and do the following steps:
+
+`mvn clean package`
+
+`mvn boost:docker-build`
+
+NOTE: The docker-build goal creates a Liberty specific Dockerfile in the root folder of the Spring Boot project
+in the absence of a Dockerfile and builds the docker image.
+
+
+=== Push Docker image
+
+In order to push an image to a public or private registry, you need to configure the `repository` parameter
+in the format `[REGISTRYHOST/][USERNAME/]NAME`. The tag parameter is set to `latest` by default.
+
+You can configure the plugin as below:
+----
+<plugin>
+  <groupId>io.openliberty.boost</groupId>
+  <artifactId>boost-maven-plugin</artifactId>
+  <version>0.1</version>
+  <configuration>
+    <repository>[your_domain_name]/${project.artifactId}</repository>
+  </configuration>
+  <executions>
+    <execution>
+      <goals>
+        <goal>docker-build</goal>
+        <goal>docker-push</goal>
+      </goals>
+    </execution>
+  </executions>
+</plugin>
+----
+
+`mvn clean install` will create an image with the given `repository` name with default tag `latest` and push the image to the registry.
+
+Alternatively, you can configure the plugin with no execution goals and do the following steps:
+
+`mvn clean package`
+
+`mvn boost:docker-build`
+
+`mvn boost:docker-push`
+
+
+*Note* :
+
+For public registry like Docker Hub, replace `[your_domain_name]` with your Docker Hub username.
+
+For private registry like IBM Container registry, replace `[your_domain_name]` with with your IBM Container Registry Host. For example, `registry.ng.bluemix.net`.
+
+== Authentication required to push Docker Image
```

Comment:
The previous section titles had "image" in lowercase. This one is uppercase. Not sure which is right, but let's be consistent.